### PR TITLE
feat(client): add audit trail page

### PR DESCRIPTION
**Pull request by Mackbellemore**
#
## Changes this PR introduces

- Adds an audit trail page

## How to test this PR

- POST, PATCH, DELETE, DOWNLOAD stuff and check the audit trail page

## Notes for reviewers

- a note

## Checklist

- [x] I renamed the title of my pull request to follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary).
- [x] I have added tests to cover my changes (if applicable).

### CODE REVIEW

**Comment by Mackbellemore** [comment]
Task linked: [CU-p54bmv Audit trail page](https://app.clickup.com/t/p54bmv)

**Review by gordonpn — APPROVED** [review]
Wow this is great

File: client/src/components/AuditTrail/AuditTrailTable.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          </Tr>
+        </Thead>
+        <Tbody>
+          {isSuccess &&
+            data.data
+              .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+              .map((log, index) => (
+                <Fragment key={index}>
+                  <AuditTrailTableRow log={log} />
+                </Fragment>
```
Comment (gordonpn) [inline client/src/components/AuditTrail/AuditTrailTable.jsx:47]: I believe you can move the key prop from `Fragment` to `AuditTrailTableRow` and get rid of the fragment. 

Only edit if you're pushing another commit on this branch tho. (Not a big deal)

**Review by gordonpn — APPROVED** [review]

File: client/src/components/AuditTrail/AuditTrailTable.jsx (hunk 1 above)
Comment (Mackbellemore) [reply client/src/components/AuditTrail/AuditTrailTable.jsx:47]: Ill come back for this, ill yoink karins search bar later on

**Review by DavidLiang01 — APPROVED** [review]

**Pull request merged**